libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ModelOverview.tsx

```diff
@@ -327,6 +351,33 @@ export class ModelOverview extends React.Component<
                   .helpMeChooseMetricsButton
               }
             </ActionButton>
+            {this.context.dataset.task_type === DatasetTaskType.ObjectDetection && (<ComboBox
```

Comment:
Could you create a new child component for these 3 widgets? This can help resolve the lint render complexity issue